pod/perldelta.pod

```diff
@@ -147,6 +147,13 @@ L<XXX> has been upgraded from version A.xx to B.yy.
 
 XXX If there was something important to note about this change, include that here.
 
+=item *
+
+L<podlators> has been upgraded from version 5.01 to v6.0.2.  The distribution
+now uses semantic versioning for the package and module versions, with a C<v>
+prefix to work with Perl's packaging system.  C<Pod::Man>, in particular, has
```

Comment:
I'm not sure about the POD here. `L<podlators>` looks wrong to me because podlators is not a module name (and you can't `perldoc podlators`, for example).
On the other hand, `C<Pod::Man>` should probably be `L<Pod::Man>`.

Also, I'd start a new paragraph after "... to v6.0.2", like in the template.

Reply:
With regard to your first sentence ... we don't have a consistent style within perldelta.
```
pod/perl5320delta.pod
532:L<podlators> has been upgraded from version 4.11 to 4.14.
1427:podlators,

...

pod/perl5380delta.pod
709:L<podlators> has been upgraded from version 4.14 to 5.01.

...
pod/perl5280delta.pod
978:The C<podlators> distribution has been upgraded from version 4.09 to 4.10.

...
pod/perl5300delta.pod
549:podlators has been upgraded from version 4.10 to 4.11.

...
pod/perl5400delta.pod
587:L<podlators> has been upgraded from version 5.01 to 5.01_02.
```
But in each case above, it's the CPAN distribution name that has been used.

Reply:
The automated tools unfortunately don't account properly for dists where the name doesn't match the primary module. But it's wrong to link to the dist name like that.

Reply:
Please re-review.